internal/collector/collector.go

```diff
@@ -223,15 +242,18 @@ func (f *Factory) NewCollectorContainer(secretNames []string, clusterID string,
 	return &collector
 }
 
-func (f *Factory) ReconcileInputServices(er record.EventRecorder, k8sClient client.Client, namespace, selectorComponent string, owner metav1.OwnerReference, visitors func(o runtime.Object)) error {
+func (f *Factory) ReconcileInputServices(er record.EventRecorder, k8sClient client.Client, namespace, name, selectorComponent string, owner metav1.OwnerReference, visitors func(o runtime.Object)) error {
 	if f.CollectorType != logging.LogCollectionTypeVector {
 		return nil
 	}
 
 	for _, input := range f.ForwarderSpec.Inputs {
 		if input.Receiver != nil && input.Receiver.HTTP != nil {
 			listenPort := input.Receiver.HTTP.GetPort()
-			if err := network.ReconcileInputService(er, k8sClient, namespace, input.Name, selectorComponent, input.Name, listenPort, listenPort, owner, visitors); err != nil {
+			// Make input service name more unique
+			// Format: inputName-Namespace-CLFName
+			serviceName := fmt.Sprintf("%s-%s-%s", input.Name, namespace, name)
```

Comment:
> Multiple CLFs with the same HTTP input receiver

I can see where "input.name" won't work in the same NS but we should be able to use "input.name" + clf.name for uniqueness.  Consider how you ref a service which has to be routable .  The full name would mean you would include the NS twice(e.g. input-namespace-clfname.namespace.svc.cluster.local); this seems clunky.  I would consider the pattern we use for other objects and suggest:

<clf.name>-<input.name> which should make it unique within the NS, unless I'm forgetting something